share/models/fields.py

```diff
@@ -24,6 +26,8 @@
 
 from db.deletion import DATABASE_CASCADE
 
+SENSITIVE_DATA_KEY = jwe.kdf(settings.SECRET_KEY.encode('utf-8'), settings.SALT.encode('utf-8'))
```

Comment:
Can you move this into `settings.py`? Generally speaking you'll want to keep constant values in `settings.py`